### DISCUSSION TITLE
How should i use v2.2.9's new feature partition_key for search？

### DISCUSSION THREAD

**Discussion by suparek** [milvus-io/milvus#24733] [Q&A and General discussion] [answered]
I can't find a example for search using partition_key, How should i use v2.2.9's new feature partition_key for search？

**Comment by xiaofan-luan** [comment]
> I can't find a example for search using partition_key, How should i use v2.2.9's new feature partition_key for search？

Seem as scalar filtering.
using partitionkey = xxx or partition key in [xxx,yyy]

**Reply by suparek** [reply]
use expr ？

**Reply by xiaofan-luan** [reply, marked as answer]
> use expr ？

exactly.
Milvus will help you to redirect to right partition and filtering the result

**Reply by suparek** [reply]
Thank you very much!
